source/board/STM32/STM32F7/STM32F769IDISCO/Kconfig-boardChoices

```diff
@@ -0,0 +1,25 @@
+#
+# file: Kconfig-boardChoices
+#
+# author: Copyright (C) 2014-2017 Kamil Szczygiel http://www.distortec.com http://www.freddiechopin.info
+# author: Copyright (C) 2017 Cezary Gapinski cezary.gapinski@gmail.com
+# author: Copyright (C) 2017 elbe informatik GmbH
```

Comment:
If you (or anyone else in the future) regenerate this file with the script, the last copyright will be gone... Let me know if this is a problem for you.